File: src/main/java/com/wezaam/withdrawal/service/PaymentMethodService.java
package com.wezaam.withdrawal.service;

import com.wezaam.withdrawal.model.PaymentMethod;
import com.wezaam.withdrawal.repository.PaymentMethodRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class PaymentMethodService {

	@Autowired
	PaymentMethodRepository paymentMethodRepository;

	public PaymentMethod findById(Long id){
		return paymentMethodRepository.findById(id).orElseThrow();
	}
}

File: src/main/java/com/wezaam/withdrawal/service/PaymentMethodService.kt
package com.wezaam.withdrawal.service

import com.wezaam.withdrawal.repository.PaymentMethodRepository
import org.springframework.stereotype.Service

@Service
class PaymentMethodService(private val paymentMethodRepository: PaymentMethodRepository) {

    fun findById(paymentMethodId: Long) = paymentMethodRepository.findById(paymentMethodId)

}